using-timescaledb/reading-data.md

```diff
@@ -147,6 +147,26 @@ SELECT
   ORDER BY time
 ```
 
+### Delta [](rate)
+
+In many monitoring (and IoT) use cases, devices or sensors report metrics that typically do not
+change. In cases where the value changes, these changes are considered anomalies. When querying
+for these changes in values over time, users typically do not want to pull all the values where
+no changes were observed since that impacts the performance of a query.
+
+Users can leverage a combination of window functions and subselects to achieve this. The below query
+uses diffs to filter rows where values have not changed, but continues to pull rows where
```

Comment:
i would change the last sentence to 
`but only transmits row where values have changed.`

Reply:
Done